Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
```

Comment:
btw: why do you need "below" at all?   If the input is adjacent this is just noise to me - does not matter if it's directly below or directly next to it,  a colon ":"  should suffice IMHO